wire/error.go

```diff
@@ -9,6 +9,153 @@ import (
 	"fmt"
 )
 
+// ErrorCode describes a kind of message error.
+type ErrorCode int
+
+// These constants are used to identify a specific Error.
+const (
+	// errDefault is the default error code. It is not intended to be used
+	// explicitly as an error code.
+	// errDefault ErrorCode = iota
+	// ErrNonCanonicalVarInt is returned when a variable length integer
+	// is encoded non-canonically.
+	ErrNonCanonicalVarInt ErrorCode = iota + 1
+	// ErrVarStringTooLong is returned when a variable string exceeds the
```

Comment:
Please add a newline before each error + comment block to be consistent with the rest of the code base.